tests/e2e/init-node.sh

```diff
@@ -57,7 +57,8 @@ evmosd config set client keyring-backend "$KEYRING"
 echo "$VAL_MNEMONIC" | evmosd keys add "$VAL_KEY" --recover --keyring-backend "$KEYRING" --algo "$KEYALGO"
 
 # Store the validator address in a variable to use it later
-node_address=$(evmosd keys show -a "$VAL_KEY")
+NODE_ADDRESS=$(evmosd keys show -a "$VAL_KEY")
+export NODE_ADDRESS
```

Comment:
I think this can be removed altogether rather - anyone who's using the script will probably re-export the name and not know that there is a variable called `$NODE_ADDRESS`

Reply:
Ok nice, I added the export because I was not sure about the external usage, but better if we can remove it